### ISSUE TITLE
List index out of range error when computing novelty

### ISSUE THREAD

**Issue by maximusunc** [TranslatorSRI/answer-appraiser#21]
https://github.com/TranslatorSRI/answer-appraiser/blob/1040864694a24dbcf426dafa5452926361cad32a/app/novelty/extr_smile_molpro_by_id.py#L43

**Comment by maximusunc** [comment]
Here's the full traceback:
```text
[2024-04-11 14:28:50,017: ERROR/appraiser.d5d8ba66]: Traceback (most recent call last):
  File "/app/app/novelty/compute_novelty.py", line 421, in compute_novelty
    similarity_map = await molecular_sim(known, unknown, message)
  File "/app/app/novelty/compute_novelty.py", line 63, in molecular_sim
    smile_known = await mol_to_smile_molpro(known_ids)
  File "/app/app/novelty/extr_smile_molpro_by_id.py", line 43, in mol_to_smile_molpro
    smiles[data_mol[i]] = smile
IndexError: list index out of range
```

**Comment by pg427** [comment]
I've accounted for this issue in the updated novelty code. Can you tell me if this issue reoccurs ?

**Comment by maximusunc** [comment]
Looks like it's fixed in #30, but I'll reopen if I see it again!
